examples/serial.c

```diff
@@ -0,0 +1,41 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <stdint.h>
+#include <pthread.h>
+
+#include "wiringX.h"
+
+typedef struct wiringXSerial_t {
+	unsigned int baud;
+	unsigned int databits;
+	unsigned int parity;
+	unsigned int stopbits;
+	unsigned int flowcontrol;
+} wiringXSerial_t;
```

Comment:
Fixed. 
When compling , I just linked the lib and didn't cpack and install wiringX. Seems that structure defined in wiringX.h can't be found if so.